src/util/bin/cosi-graph-update.js

```diff
@@ -17,6 +17,7 @@ const Graph = require(path.join(cosi.lib_dir, 'graph'));
 app.
     version(cosi.app_version).
     usage('[options] <config_file>').
+    description('Update Circonus graph from local graph config file (/opt/circonus/cosi/registration/registration-*.json)').
```

Comment:
Right. I just wanted to point the user at an example of a valid config file, which they could use to get started. Maybe `config-*` is better?